src/main/resources/view/MainWindow.fxml

```diff
@@ -46,6 +47,19 @@
           </padding>
         </StackPane>
 
+        <HBox id="contactList" fx:id="contactListPane" styleClass="pane-with-border" minWidth="340" prefWidth="340">
+            <!-- Contact list -->
+            <VBox fx:id="personList" styleClass="pane-with-border" VBox.vgrow="ALWAYS" minWidth= "400" prefWidth="400">
```

Comment:
The outer container has a `minWidth` of `340` but this has a `minWidth` of `400`?